docs/source/faq.rst

```diff
@@ -332,3 +332,55 @@ Note that the above examples are similar to running the garbage collector inside
 
     :class:`~optuna.integration.ChainerMNStudy` does currently not provide ``gc_after_trial`` nor callbacks for :func:`~optuna.integration.ChainerMNStudy.optimize`.
     When using this class, you will have to call the garbage collector inside the objective function.
+
+How do I suggest variables which represent the proportion, that is, are in accordance with Dirichlet distribution?
+------------------------------------------------------------------------------------------------------------------
+
+When you want to suggest `n` variables which represent the proportion, that is, `p[0], p[1], ..., p[n-1]` which satisfy `0 <= p[k] <= 1` for any `k` and `p[0] + p[1] + ... + p[n-1] = 1`, try the following.
+These variables are in accordance with the flat Dirichlet distribution.
```

Comment:
To make the part more informative, can we add a link to the page of the Dirichlet distribution of Wikipedia https://en.wikipedia.org/wiki/Dirichlet_distribution? I suppose the Dirichlet distribution is not so common for practioners.